client/components/Interfaces.tsx

```diff
@@ -1,10 +1,10 @@
 export interface ProjectQuery {
```

Comment:
Left this comment on #55 but this should be renamed to a type definition file and moved to the @types directory.

Reply:
Everything from Interfaces?